src/lib/button/_button-theme.scss

```diff
@@ -99,6 +99,23 @@
     @include _mat-button-ripple-color($theme, default, 0.1);
   }
 
+  .mat-stroked-button {
+    color: rgba(0, 0, 0, 0.87);
```

Comment:
This should probably be pulled from the foreground palette (which accounts for light vs. dark theme)